### DISCUSSION TITLE
More Newbie Help

### DISCUSSION THREAD

**Discussion by BobbyPr** [missionpinball/missionpinball#105] [Q&A]
Coil not firing.
config file opens.
Switch seems to work.
i have continuity from negative side of coil to pin 0-0-04 on jumper 7  
Coil does work when i hit it with power directly (out outside of game).

[2023-07-16-17-44-11-mpf-DESKTOP-8DSP8H2.log](https://github.com/missionpinball/missionpinball/files/12065185/2023-07-16-17-44-11-mpf-DESKTOP-8DSP8H2.log)
[config.txt](https://github.com/missionpinball/missionpinball/files/12065194/config.txt)

**Comment by toomanybrians** [comment]
You’ll need to provide more details. What are you doing to fire the coil? What have you tried? What doesn’t work? Are you able to fire the coil outside of MPF? (e.g. is it an MPF issue or a hardware / wiring issue?)

Also I can see from your configs that you’re using OPP hardware. This discussion group is focused on the MPF codebase and MPF itself. The specifics of how to use each hardware platform and wiring, etc. isn’t covered here. You may want to see where else people talk about OPP? (I think I’ve seen cobrapin stuff in the mpf-users google group? Maybe try there?)

Of course feel free to post here as well, but I’ve never seen any OPP discussion here so you might not get a response.

**Comment by worldpeace-germany** [comment]
I could guess that it might not be enabled. Coils are only get enabled once the game has started. It is somewhere mentioned in the docs :-)

```
flippers:         # combine swichtes and coils to have a flipper finger
  left_flipper:
    main_coil: c_flipper_left_main
    hold_coil: c_flipper_left_hold
    activation_switch: s_left_flipper
    enable_events: machine_reset_phase_3
```

You might not reach that state, what I did for simple test is to specify the enable event. I see in your log that this event is being fired. Can you try this?
